test/testcommon/src/main/java/io/pravega/test/common/JwtBody.java

```diff
@@ -20,7 +20,7 @@
 @Builder
 @Getter
 @Setter
-class JwtBody {
+public class JwtBody {
```

Comment:
This is such a small amount of json, and it doesn't even have nesting. Rather than dealing with dependency managment of Gson can we just use a Format template?

Reply:
This is an existing piece of code and I think we should explore seeing if that'd make sense and do the change separately of this PR.